cmd/rootMain.go

```diff
@@ -43,6 +43,36 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
 	// Configure logrus to emit simple text
 	log.SetFormatter(&log.TextFormatter{DisableTimestamp: true})
 
+	var execfile string
+	if len(args) > 0 {
+		execfile = args[0]
+	}
+
+	// Set plugin interactivity to false if execfile or rootCommandFlag were specified.
+	// Also set to false if rootVerifyInstallFlag so we don't try to fork right before running
+	// verification; verify-install should have no interactive behavior.
+	plugin.InitInteractive(execfile == "" && rootCommandFlag == "" && !rootVerifyInstallFlag)
```

Comment:
Sorry, one other nit. The comment talks about the "false" conditions, but the code talks about the "true" conditions. This creates a mis-match in my head when I'm trying to understand why this is correct. Could the comment instead talk about the "true" condition? So something like
```
interactivity is true if a script (execfile) isn't specified, a command isn't specified, and verifyInstall is false
```

Reply:
Done.